packages/example/src/components/AccountInfo/AccountInfo.tsx

```diff
@@ -40,60 +40,37 @@ export function AccountInfo(): JSX.Element {
           />
         </Col>
         <Col span={6}>
-          <Statistic title="Status" value={'syncing'} />
+          <Button type="default" size="large" onClick={() => void getBalance()}>
+            {balance === null ? 'Get Balance' : 'Refresh Balance'}
+          </Button>
         </Col>
       </Row>
       <Divider />
       <Form {...formLayout}>
         <Form.Item label="Address" colon={false}>
           <Input
             size="large"
-            placeholder="Signature"
+            placeholder="loading address..."
             value={address}
             addonAfter={<CopyButton data={address} />}
             disabled={!address}
           />
         </Form.Item>
         <Form.Item label="View Key" colon={false} style={{ marginBottom: 0 }}>
-          <Form.Item
-            name="wk"
-            style={{ display: 'inline-block', width: 'calc(100% - 88px)' }}
-          >
-            <Input
-              size="large"
-              placeholder="Signature"
-              value={'pkaaaa'}
-              addonAfter={<CopyButton data={'signature'} />}
-              disabled
-            />
-          </Form.Item>
-          <Form.Item
-            name="wkb"
-            style={{ display: 'inline-block', width: '80px', margin: '0 0 0 8px' }}
-          >
-            <Button size="large" style={{ width: 80 }} onClick={() => {}}>
-              Show
-            </Button>
-          </Form.Item>
-        </Form.Item>
-        <Form.Item label="Private Key" colon={false} style={{ marginBottom: 0 }}>
-          <Form.Item
-            name="pk"
-            style={{ display: 'inline-block', width: 'calc(100% - 88px)' }}
-          >
+          <Form.Item style={{ display: 'inline-block', width: 'calc(100% - 88px)' }}>
             <Input
               size="large"
-              placeholder="Signature"
-              value={'pkaaaa'}
-              addonAfter={<CopyButton data={'signature'} />}
-              disabled
+              placeholder="Hidden View Key"
```

Comment:
Suggestion: you can improve UI by changing `Hidden View Key` with 53 starts (`*`)

```suggestion
              placeholder="*****************************************************"
```

In that way, you created an illusion that the key is real there but is hidden